docs/Syntax.md

```diff
@@ -69,7 +69,7 @@ The grammar below describes the syntax:
    graph ::= id value-info-list '=>' value-info-list node-list
    other-data ::= id ':' value
    other-data-list ::= '<' other-data (',' other-data)* '>'
-   fun-attr-list ::= '<' id-list '>'
+   fun-attr-list ::= '<' id | attr (, id | attr) '>'
```

Comment:
What about str for attributes without values?

Reply:
A * is missing after the )

Reply:
is that this line for?
repeated string attribute = 6;
and in such case, it will be:
fun-attr-list ::= '<' id (, id)* '>'
which is a subset of the one defined at line 72.

Reply:
It used to be for "repeated string attribute". Now, it is for both that line and the "repeated AttributeProto" you added.

Reply:
The `,` should be quoted (`','`) for the 2+ case:

```
fun-attr-list ::= '<' id | attr (',' id | attr) '>'
```